Contribute/index.md

```diff
@@ -0,0 +1,140 @@
+# Contribute to Umbraco Documentation
+
+This guide will cover the following sections:
+ - [Ways of contributing](#ways-of-contributing)
+ - [Markdown conventions](#markdown-conventions)
+ - [Contribution badge](#contribution-badge)
+ - [Documentation Curators](#documentation-curators)
+
+ ## Ways of contributing
+This guide will cover how to contribute to the Umbraco Documentation, if you are interested in contributing to the Umbraco CMS you can find out how in their [contribution guidelines](https://github.com/umbraco/Umbraco-CMS/blob/dev-v7/.github/CONTRIBUTING.md).
+
+There are two ways to contribute to the Umbraco Documentation:
+- [Creating an issue](#creating-an-issue)
+- [Creating a pull request](#creating-a-pull-request)
+
+### Creating an issue
+The Umbraco Documentation uses [Github issues]((https://github.com/umbraco/UmbracoDocs/issues)) to manage issues with the documentation.
+You are very welcome to create an issue if you feel some documentation is missing or wrong, the issue board is also used for discussions about possible improvements or ways to deal with something in the documentation. You can also find a button in the top right corner of every page in the documentation itself that looks like this:
+![Our issue button](images/report-issue.png)
+
+#### Issue labels 
+The issue repository has several labels, these are not something you can add yourself but they may be added to your issue and other issues. Here is a quick overview of the commonly used labels and what they mean:
+
+- `discuss` - this label is added to issues that are not a tangible problem or very large, an issue with the discuss label is something everyone is encouraged to give their opinion on!
+- `missing documentation` - for issues about missing information, if you see this and wish to make a pull request on it please write that in a comment
+- `incorrect documentation` - for documentation that is wrong, this can be due to many things, often it is outdated
+- `up-for-grabs` - any issue with an up-for-grabs label is something noone is actively working on and is open for anyone wishing to contribute
```

Comment:
up-for-grabs is mostly **curated tasks specifically for new contributors** 

(c) https://up-for-grabs.net/